qa/pytest_komodo/cc_modules/test_token.py

```diff
@@ -53,8 +53,11 @@ def test_token(test_params):
 #        assert result == []
 
     # trying to create token with negative supply
-    result = rpc.tokencreate("NUKE", "-1987420", "no bueno supply")
-    assert_error(result)
+    try:
+        result = rpc.tokencreate("NUKE", "-1987420", "no bueno supply")
+        assert_error(result)
+    except RPCError as e:
```

Comment:
Could you please use assert for excpetions excepting https://docs.pytest.org/en/latest/reference.html#pytest-raises ?

https://stackoverflow.com/questions/23337471/how-to-properly-assert-that-an-exception-gets-raised-in-pytest there you can find descriptive answer why current implementation is less infromative